docs/get_started/README.md

```diff
@@ -45,3 +45,11 @@ The [Kubernetes CLI (`kubectl`)](https://kubernetes.io/docs/tasks/tools/install-
    helm install kserve-crd oci://ghcr.io/kserve/charts/kserve-crd --version v0.13.0
    helm install kserve oci://ghcr.io/kserve/charts/kserve --version v0.13.0
    ```
+
+   This second method requires cert-manager to be installed in the cluster. If you don't have cert-manager installed, you can install it with:
```

Comment:
The first method also requires cert manager but it was installed as part of the script, we can probably change quick install to support the raw deployment mode and change to use helm for installing kserve.

Reply:
cc @sivanantha321 @andyi2it

Reply:
Yeah! I will raise a PR for that.

Reply:
Does that mean this PR is not needed any more?

Reply:
PING

Reply:
PING

Reply:
I raised a PR here. https://github.com/kserve/kserve/pull/3813

Reply:
IMO, We can remove the helm installation here (as this docs is only for getting started and try out KServe ) and add it in the administrator guide which has step by step process for installing kserve with its prerequisites. There you can add the helm installation option. https://kserve.github.io/website/latest/admin/serverless/serverless/